src/validationUtils.ts

```diff
@@ -101,3 +102,53 @@ export function validateVATNumber(vatNumberInput: string | number): boolean {
 
   return valid;
 }
+
+/**
+ * Validates if a given phone number is a Greek mobile phone number.
+ *
+ * @param {string} mobilePhone - The mobile phone number to validate.
+ * It may include special characters like spaces, dashes, parentheses, or dots, which will be ignored during validation.
+ *
+ * @returns {boolean} - Returns `true` if the phone number is a valid Greek mobile number; otherwise, `false`.
+ */
+export function isValidMobilePhone(mobilePhone: string): boolean {
+  const mobilePhoneRegex = RegExp(/^(\+30|0030)?69\d{8}$/);
+
+  return mobilePhoneRegex.test(mobilePhone.replace(/[\s\-().]/g, ""));
```

Comment:
Let's keep it as simple as possible.

```suggestion
  let mobilePhoneStr = mobilePhone;

  if (typeof mobilePhone === "number")  {
    mobilePhoneStr = String(num).slice(-10)
  }
 
  const mobilePhoneRegex = RegExp(/^(\+30|0030)?69\d{8}$/);

  return mobilePhoneRegex.test(mobilePhoneStr.replace(/[\s\-().]/g, ""));
```

Reply:
Hey @tsevdos!

Numbers starting with 2 zeros are not accepted at all.

Since either way we use string type to apply logic for each method then I decided first things first to convert to string using the respective constructor.

Removed any RegExp constructors to simplify code even further and kept it only in the case of any template literals.